src/sentry/static/sentry/app/views/organizationDiscover/data.jsx

```diff
@@ -19,6 +19,11 @@ export const PROMOTED_TAGS = [
   {name: 'tags[sentry:release]', type: 'string'},
 ];
 
+// All tags are assumed to be strings, except the following
+export const SPECIAL_TAGS = {
+  'tags[os_rooted]': TYPES.BOOLEAN,
```

Comment:
Are users going to be able to make their own non-string tags?

Reply:
No, there isn't a way for users to do this. All of the built-in (non user defined) tags are also currently all strings except for this one exception